### DISCUSSION TITLE
Which of the seven continents is the only one that is a single country?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#600] [Galaxy Trivia] [answered]
*answer with the flag of the country*

**Comment by peterbaccaro** [comment]
🇦🇺 

Australia is the only continent on Earth that is also a single country. The continent of Australia is made up of the mainland of the country of Australia, as well as the island of Tasmania and several smaller islands.

**Comment by thuongtruong1009** [comment]
The continent that is the only one consisting of a single country is Australia, also known as the Australian continent or Oceania.

Here is the flag of Australia: 🇦🇺

**Comment by Veercodeprog** [comment]
Australia : The king of Kangaroos Country is here for you to be the only country that is continent as well.

**Comment by Meriem-BM** [comment, marked as answer]
🇦🇺

**Reply by ParanoidUser** [reply]
Welcome to the galaxy trivia party, @Meriem-BM! 🍀🤞
